susecloud

```diff
@@ -379,3 +379,12 @@ if [ -e /opt/dell/crowbar_framework/.crowbar-installed-ok ]; then
   find_and_plog_files_0 $chef_export_dir -type f
   rm -rf $chef_export_dir
 fi
+
+###############################################################
+section_header "RabbitMQ broker"
+if [ -e /usr/sbin/rabbitmqctl ]; then
+  rabbitmqctl report > /var/log/rabbitmq/rabbitmqctl_report.log
```

Comment:
Nitpick: `/var/log` is supposed to be for log files which persist, so it doesn't really make sense to create a file under it and then remove it shortly after.  If this file is really intended to be a temporary file, then it should be created as one via `mktemp`.  This would avoid the tiny risk of two supportconfigs running at the same time and interfering with each other.

Also, I would prefer using a temporary variable for the filename, rather than repeating a long path 3 times.  DRY!